Sources/MusicXML/Decoding/Decoding.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 import XMLCoder
 
 extension Score {
+    public static let topLevelTagKey = "topLevelTagKey"
```

Comment:
Would it make sense to instead store `let traversalDirection: CodingKeys` in `Score`, bypassing the `userInfo` layer? I am also thinking this property might be helpful when we get to encoding the `Score` value by a similar pattern?

Reply:
That doesn't work - `self` hasn't been initialized yet. 

The only way I pass information to `public init(from decoder: Decoder) throws` is thru the `decoder`.